fvm/handler/contract.go

```diff
@@ -98,14 +99,31 @@ func (h *ContractHandler) RemoveContract(address runtime.Address, name string, s
 	return nil
 }
 
+type contractUpdateList []programs.ContractUpdate
+
+func (a contractUpdateList) Len() int      { return len(a) }
+func (a contractUpdateList) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a contractUpdateList) Less(i, j int) bool {
+	return a[i].Address.String() < a[j].Address.String() ||
+		(a[i].Address.String() == a[j].Address.String() && a[i].Name < a[j].Name)
+}
```

Comment:
Maybe avoid (repeated) string allocation by comparing address arrays directly, and improve naming of receiver:

```suggestion
func (l contractUpdateList) Len() int      { return len(l) }
func (l contractUpdateList) Swap(i, j int) { l[i], l[j] = l[j], l[i] }
func (l contractUpdateList) Less(i, j int) bool {
	switch bytes.Compare(l[i].Address[:], l[j].Address[:]) {
	case -1:
		return true
	case 0:
		return l[i].Name < l[j].Name
	default:
		return false
	}
}

```